src/wmtk/operations/tri_mesh/VertexLaplacianSmooth.cpp

```diff
@@ -35,7 +34,7 @@ bool VertexSmooth::before() const
     return true;
 }
 
-bool VertexSmooth::execute()
+bool VertexLaplacianSmooth::execute()
```

Comment:
after line 47 (the star invocation) should just call the parent class's implementation